.github/ISSUE_TEMPLATE/new_metapackage.yml

```diff
@@ -0,0 +1,114 @@
+  - type: input
+    id: pkg_name
+    attributes:
+      label: Package Name
+      description: |
+        The convention is to use lowercase names with the following format: `toolname` or `tool-name`. For example `capa` and `fakenet-ng`. **Please add this name to the issue title as well.**
+      placeholder: ex. fakenet-ng
+    validations:
+      required: true
+  - type: input
+    id: tool_name
+    attributes:
+      label: Tool Name
+      description: The name of the tool being installed (usually the file name with the `.exe`), normally different from the package name. For example fakenet-ng (package name) vs FakeNet-NG (tool name).
+      placeholder: ex. FakeNet-NG
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: Tool's version number
+      placeholder: ex. 1.4.11
+    validations:
+      required: true
+  - type: dropdown
+    id: category
+    validations:
+      required: true
+    attributes:
+      label: Category
+      description: Which category should this tool be installed to?
+      options:
+        - Android
+        - Debuggers
+        - Delphi
+        - Developer Tools
+        - Disassemblers
+        - dotNet
+        - Flash
+        - Forensic
+        - Hex Editors
+        - Java
+        - Javascript
+        - Net
+        - Office
+        - PDF
+        - Pentest
+        - PowerShell
+        - Python
+        - Text Editors
+        - Utilities
+        - VB
+        - Web Application
+  - type: input
+    id: authors
+    attributes:
+      label: Tool's authors
+      description: Comma separated list of authors for tool
+      placeholder: ex. Elmo, Bert, Grover, Cookie Monster
+    validations:
+      required: true
+  - type: input
+    id: description
+    validations:
+      required: true
+    attributes:
+      label: Tool's description
+      description: The description should be short and not include version specific details or other information that is likely to change in a future version.
+      placeholder: ex. FakeNet-NG is a next generation dynamic network analysis tool for malware analysts and penetration testers.
+  - type: dropdown
+    id: type
+    validations:
+      required: true
+    attributes:
+      label: Package type
+      description: This is needed to know how to send the automatic PR.
+      options:
+        - METAPACKAGE
+  - type: input
+    id: dependency
+    validations:
+      required: true
+    attributes:
+      label: Dependency
+      description: Name of the package to install via dependencies.
+  - type: input
+    id: shim_path
+    validations:
+      required: true
+    attributes:
+      label: Shim path
+      description: Metapackage shim path.
```

Comment:
Should we add more details on what this is? Also an example could be helpful.

Reply:
Yes, I was not sure how to explain it. Maybe @MalwareMechanic can help me 😄

Reply:
How about:

Installed Chocolatey packages automatically create "shims" (i.e., executable proxies) to the installed applications. The default Chocolatey shim location is `%PROGRAMDATA%\chocolatey\bin`. Please provide the path for the Chocolatey "shimmed" executable (e.g., `%PROGRAMDATA%\chocolatey\bin\ollydbg.exe`).